src/components/empty-state/EmptyState.tsx

```diff
@@ -113,8 +38,28 @@ export const EmptyState: React.FC<EmptyStateProps> & {
   Image: typeof EmptyStateImage
   Title: typeof EmptyStateTitle
   Body: typeof EmptyStateBody
-} = ({ size = 'sm', ...props }) => (
-  <EmptyStateContainer size={size} {...props} />
+} = ({ size = 'sm', children, ...props }) => (
+  <EmptyStateContainer size={size} {...props}>
+    {React.Children.map(children, (child) => {
+      // add relevant check for valid child
+      if (!React.isValidElement(child)) {
+        throw new Error()
```

Comment:
it is a small change but I think we should be passing an error message here. 
```suggestion
        throw new Error("Invalid child element in EmptyState")
```

or something similar